src/main/cpp/blaze.cc

```diff
@@ -261,7 +261,7 @@ class BlazeServer final {
   // is in connected state.
   unsigned int Communicate(
       const std::string &command, const std::vector<std::string> &command_args,
-      const std::string &invocation_policy,
+      bool uses_param_file, const std::string &invocation_policy,
```

Comment:
Maybe refactor `Communicate` to take a ` const OptionProcessor *option_processor`, as the first three arguments are currently: 
```

server->Communicate(
  option_processor.GetCommand(), 
  option_processor.GetCommandArguments(), 
  option_processor.UsingParamFile()
```